app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -320,6 +322,7 @@ class BrowserFragment :
         }
 
         setSitePermissions(view)
+        initialiseMediaSessionFeature(view)
```

Comment:
Can we just in-line the feature here (similar to `topSitesFeature`, `appLinksFeature`, etc.)?